### DISCUSSION TITLE
비스팀 모드 일련번호 부여

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#542] [일반]
창작마당에 없는 모드를 등록할 때는

[문자열 생성기](https://randstrgen.lazyig.com/) 등으로 임의의 7자리 수를 생성한 뒤

앞에 NS를 붙여서 기존 모드와 중복되는지 확인 후

아래 이미지처럼 그걸 워크샵ID 대신 사용할 것

<img width="167" alt="화면 캡처 2024-03-09 000515" src="https://github.com/RimWorldKorea/RMK/assets/150018627/64a884fe-9f65-463d-83f1-96afe93dcf81">

@zilrty

**Comment by zilrty** [comment]
오우 너무 멋지고
